### PR TITLE
Fixes #36570 - pin wizard navigation to bottom

### PR DESCRIPTION
**Pull request by MariaAga**
~draft until foreman uses the pf4 `Page` component~ - done
Screenshot from 2023-07-05 12-27-56

Screenshot from 2023-07-05 12-28-02

### CODE REVIEW

File: webpack/JobWizard/JobWizard.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
+main#foreman-main-container {
+  height: 100%;
```
Comment (MariaAga) [inline webpack/JobWizard/JobWizard.scss:2]: will affect all pages in foreman

**Review by adamruzicka — APPROVED** [review]
Looks good to me, but let's keep this around for a while until the things that don't require 3.8 get in

**Comment by adamruzicka** [comment]
Alright, no need to hold this any longer, let's get this in

**Comment by adamruzicka** [comment]
Thank you @MariaAga !

**Pull request merged**